src/doc/unstable-book/src/language-features/rustc-attrs.md

```diff
@@ -0,0 +1,50 @@
+# `rustc_attrs`
+
+This feature has no tracking issue, and is therefore likely internal to
```

Comment:
```suggestion
This feature has no tracking issue, and is therefore internal to
```

Reply:
That is taken from the stub, should we change the stub too?

Reply:
I have no idea... I just feel like we can be a bit more definite in this case.